### ISSUE TITLE
Bookmarks broken in SFOS 3.0.3.1

### ISSUE THREAD

**Issue by gabrielgg** [monich/harbour-books#52]
In short, SFOS <3.0.3.1 stores bookmarks in ~/.local/share/harbour-books/internal/sailfish/home (for local storage, haven't tried SD card)
In SFOS >=3.0.3.1, Books stores/expects bookmarks in ~/.local/share/harbour-books/internal/mapper/sailfish-home

A manual copy fixed the issue, but some users may not be able to do that easily.

**Comment by monich** [comment]
Yeah, mapping paths for home partition and SD card sometimes change from release to release. That shouldn't be much of a problem, books will have to be re-scanned once again when they are opened but then things should be back to normal. IMO, the worst part is that it leaves some garbage in the filesystem.

**Comment by gabrielgg** [comment]
> That shouldn't be much of a problem, books will have to be re-scanned once again when they are opened but then things should be back to normal. 
Well, re-scanning of books works perfectly well, but you do lose the bookmarks/state of each book unless you copy the files manually to the new location.  This will probably be seen by a bug by most users.
> IMO, the worst part is that it leaves some garbage in the filesystem.
Agree :) Perhaps some sort of SFOS version dependency and migration is in order, but I'm not familiar enough with the code to suggest an approach, let alone recommend a fix myself.
